interface/usergroup/facilities_add.php

```diff
@@ -287,6 +287,45 @@ function displayAlert()
         <td><font class="mandatory">*</font><span class="text"> <?php echo xlt('Required'); ?></span></td><td>&nbsp;</td><td>&nbsp;</td>
         <td>&nbsp;</td><td>&nbsp;</td>
         </tr>
+        <tr>
+            <td>
+                <hr>
+            </td>
+        </tr>
+
+        <tr>
+            <td>
+                <label><?php xl('Mailing Address', 'e'); ?>: </label>
```

Comment:
```
<label><?php echo xlt('Mailing Address'); ?>: </label>
```

Reply:
I was going to remove those but there was other work done where they were not removed. So, I left them.

Reply:
need to still address